Fix @@NESTLEVEL is 1 unit too high

RIC06X opened the pull request:
### Description

Previously, @@NESTLEVEL in Babelfish is always 1 unit high, but in Postgres backend, the value is correct. Since we only use @@NESTLEVEL in Babelfish, we changed the value to 1 unit lower in its implementation. 

### Issues Resolved

Fixed @@NESTLEVEL is 1 too high
Task: BABEL-2810

### Test Scenarios Covered ###
* **Use case based -**
  * Tested @@NESTLEVEL value in nested procedures


### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

RIC06X committed:
[BABEL-2810] Fix @@NESTLEVEL is 1 too high

This commit fixed @@NESTLEVEL in Babelfish is 1 unit too high.

Task: BABEL-2810
Signed-off-by: Yanjie Xu <yanjiex@amazon.com>

RIC06X committed:
[BABEL-2810] Rename upgrade test and Update test results

Fix naming error for sys-nestlevel-dep and updated affected tests

Task: BABEL-2810
Signed-off-by: Yanjie Xu <yanjiex@amazon.com>

RIC06X committed:
[BABEL-2810] Update upgrade schedule file

Update schedule file and delete naming error tests

Task: BABEL-2810
Signed-off-by: Yanjie Xu <yanjiex@amazon.com>

RIC06X committed:
[BABEL-2810] Fix run sql validation tests

Task: BABEL-2810
Signed-off-by: Yanjie Xu <yanjiex@amazon.com>

RIC06X committed:
[BABEL-2810] Fix upgrade sql script

Task: BABEL-2810
Signed-off-by: Yanjie Xu <yanjiex@amazon.com>

macwoody commented on test/JDBC/input/sys-nestlevel-dep-vu-prepare.sql:
```diff
@@ -0,0 +1,31 @@
+
+CREATE PROC p1
+AS
+SELECT @@NESTLEVEL AS p1BeforeShouldBe1
```
please also add a SQL view depending on nestlevel

RIC06X committed:
[BABEL-2810] Add view tests for @@nestlevel

Task: BABEL-2810
Signed-off-by: Yanjie Xu <yanjiex@amazon.com>

RIC06X committed:
[BABEL-2810] Remove view test from vu test script

This commit removed view test from vu test script and added one sql test
script to include all the tests in vu and also view test. The later sql
test script will only get executed for JDBC.

Task: BABEL-2810
Signed-off-by: Yanjie Xu <yanjiex@amazon.com>
